lib/internal/process/next_tick.js

```diff
@@ -88,20 +109,58 @@ function setupNextTick() {
     }
   }
 
+  // TODO(trevnorris): Using std::stack is much slower here than was the
+  // Float64Array stack used in a previous implementation. Problem is the
+  // Float64Array stack was a bit brittle. Investigate how to harden that
+  // implementation and possibly reintroduce it.
```

Comment:
Sounds good. For what it’s worth, it sounds like there might be good chance of improving things by making the underlying container a pre-allocated `std::vector` or something a long those lines.